### ISSUE TITLE
`vm clone` depends on deprecated em-winrm gem

### ISSUE THREAD

**Issue by rhass** [chef-partners/knife-vsphere#250]
https://github.com/chef-partners/knife-vsphere/blob/master/lib/chef/knife/vsphere_vm_clone.rb#L812

The Gem em-winrm has been deprecated, and conflicts with the current winrm gem used by Test Kitchen.

```
Customization Complete
ERROR: Gem::ConflictError: Unable to activate em-winrm-0.7.0, because winrm-1.7.3 conflicts with winrm (~> 1.3.0)
```

**Comment by swalberg** [comment]
Thanks, there's some discussion in #247, I need to grab a few minutes to test out the fix.

**Comment by swalberg** [comment]
Since I'm here anyway :) Do you want to try #251 to see if it fixes the problem? I'll try to make sure it can clone a Windows box with no problem tomorrow before I merge.

**Comment by rhass** [comment]
I can confirm removing the line referenced works to fix the issue. I did the same thing to test a fix before I saw your comments. Nevertheless, I will also try out PR #251 just to make sure there isn't any other issues and get back to you shortly. Thanks!

**Comment by swalberg** [comment]
Fixed in v1.2.9, please update your gem. Thanks for your help in reporting the bug and verifying the fix.
